WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -6,16 +6,14 @@ def index
     respond_to do |format|
       format.html
       format.js do
-        persons = Person.joins("JOIN Countries ON countryId = Countries.id")
+        persons = Person.order(:name, :countryId)
         if params[:region] != "all"
-          persons = persons.where("countryId = :region OR continentId = :region", region: params[:region])
```

Comment:
I broke out a `Person.in_region` scope. I like it.

I'm not confident enough that it's a universal performance improvement, or that the cases are similar enough, that I want to touch `Competition.belongs_to_region`

I also missed unifying the names. I can fix it if we care. If so I prefer `.in_region` over `belongs_to_region`.

Finally, I want murder the **_Use the lambda method for multiline lambdas._** Rubocop rule I ran into here. Separate PR!